extractor/fetch_index.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env sh
```

Comment:
Just curious, have you successfully run this on a more general query with hella data?

Reply:
Negative. May need to parallelize some stuff with `multiprocessing` or `threading` or some other Python library.